common/lib/xmodule/xmodule/assetstore/assetmgr.py

```diff
@@ -49,14 +50,9 @@ class AssetManager(object):
     @contract(asset_key='AssetKey', throw_on_not_found='bool', as_stream='bool')
     def find(asset_key, throw_on_not_found=True, as_stream=False):
         """
-        Finds a course asset either in the assetstore -or- in the deprecated contentstore.
+        Gets course asset in the deprecated contentstore.
+        This method was previously searching course asset either in the assetstore -or- in the deprecated contentstore.
```

Comment:
@doctoryes @alawibaba @macdiesel:

N.B. If/when we get to the point of actually storing assets elsewhere, we're going to have to change this back again.

Also, from a long-term direction standpoint, I think what needs to change is `StaticContentServer`. Ideally, we'd want all of our asset stores to act like S3: that is, we generate a URL for them on the way out that encodes any of the necessary permissions, and then when the request comes back in, all it does is a minimal permissions check, and load the asset (rather than having to look up metadata in the database).